packages/services/stripe-billing/src/router.ts

```diff
@@ -0,0 +1,73 @@
+import { FastifyLoggerInstance, reportReadiness } from '@hive/service-common';
+import { createFetchAPIHandler } from '@valu/trpc-fetch-api-adapter';
+import { Router } from 'itty-router';
+import { createLogger } from '@hive/service-common';
+import { Context, stripeBillingApiRouter } from './api';
+import { env } from './environment';
+import { createConnectionString } from '@hive/storage';
+import { createStripeBilling } from './billing-sync';
+
+const stripeBillingRouter: Router = Router();
+
+const logger = createLogger(env.log.level) as FastifyLoggerInstance;
+
+const {
+  readiness: stripeBillingReadiness,
+  start: startStripeBilling,
+  stop: stopStripeBilling,
+  stripeApi,
+  postgres$,
+  loadStripeData$,
+} = createStripeBilling({
+  logger,
+  stripe: {
+    token: env.stripe.secretKey,
+    syncIntervalMs: env.stripe.syncIntervalMs,
+  },
+  rateEstimator: {
+    endpoint: env.hiveServices.usageEstimator.endpoint,
+  },
+  storage: {
+    connectionString: createConnectionString(env.postgres),
+  },
+});
+
+const context: Context = {
+  storage$: postgres$,
+  stripe: stripeApi,
+  stripeData$: loadStripeData$,
+};
+
+const respondWithTRPC = createFetchAPIHandler({
+  router: stripeBillingApiRouter,
+  createContext: async (): Promise<Context> => context,
+});
+
+stripeBillingRouter.all('/trpc/:path+', async (req: any) => {
```

Comment:
```suggestion
stripeBillingRouter.all('/trpc/:path+',  createFetchAPIHandler({
  router: stripeBillingApiRouter,
  createContext: async (): Promise<Context> => context,
}))
```

Reply:
We can simplify the code like that

Reply:
Oh nice! I didn't know you could do that. I just followed the example there: https://github.com/esamattis/trpc-cloudflare-worker/blob/master/src/worker.ts#L3